st2common/st2common/persistence/base.py

```diff
@@ -120,7 +120,7 @@ def insert(cls, model_object, publish=True, dispatch_trigger=True,
             model_object = cls._get_impl().insert(model_object)
         except NotUniqueError as e:
             if log_not_unique_error_as_debug:
-                LOG.debug('Conflict while trying to save in DB.', exc_info=True)
+                LOG.debug('Conflict while trying to save in DB: %s.' % (str(e)))
```

Comment:
Yeah, we have been going back and forth on comma and % (I'm personally for %, but I know why you prefer comma) :P